cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md

```diff
@@ -1,5 +1,49 @@
+The [CMS Collaboration at CERN](https://cms.cern/) is pleased to announce the release of the third batch of high-level open data from the CMS detector at the Large Hadron Collider (LHC), available on the [CERN Open Data portal](http://opendata.cern.ch/). This batch contains over 550 terabytes, or 11.6 [inverse femtobarns](https://en.wikipedia.org/wiki/Barn_(unit)#Inverse_femtobarn) (fb<sup>&minus;1</sup>), of [proton-proton collision data recorded in 2012 at a centre-of-mass energy of 8 TeV](http://opendata.cern.ch/search?experiment=CMS&year=2012&type=Dataset&subtype=Collision) &mdash; around half the data from that year. Around 510 terabytes of Monte Carlo simulation data are also being made available now, with more in the data-transfer pipeline. This release includes datasets that were used to [discover the Higgs boson](https://cms.cern/physics/higgs-boson/observation-new-particle-mass-125-gev) and the data are available in the same format that CMS scientists use for research. CMS is also providing smaller, simplified datasets for use in educational contexts. As previously, the data are released into the public domain under the Creative Commons CC0 waiver.
 
 <p align="center">
 <img src="https://cms-docdb.cern.ch/cgi-bin/PublicDocDB/RetrieveFile?docid=13467&filename=CMSOpenData2012Event.png" width="70%"><br />
 <em>A collision event recorded by CMS in 2012 and available in the latest release of CMS Open Data. File for visualisation prepared by Tom McCauley</em></p>
+
+[LHC data are complicated and they are big.](https://cms.cern/news/lhc-data-be-made-public-open-access-initiative) The LHC began operations in 2010 and since then, under nominal conditions, intense bunches of protons collide inside the CMS detector up to 40 million times every second. CMS researchers have recorded petabytes of data from these collisions and have so far published [hundreds of scientific papers](http://cms.cern/news/cms-publishes-700th-research-paper) with these data. “Our data are an important element of the CMS Collaboration’s rich scientific legacy,” says CMS Spokesperson, Joel Butler. “We would like to ensure that they are not only preserved in the long run but are also available to the public, so that both CMS members and external researchers can re-examine them in the future. This is part of our [commitment to openness and long-term data preservation \[PDF\]](https://cms-docdb.cern.ch/cgi-bin/PublicDocDB/RetrieveFile?docid=6032&filename=CMSDataPolicy.pdf).”
```

Comment:
The `PDF` part does not display nicely, I'll fix it while merging...

Reply:
Noted. Sorry about that. Wanted to make sure the `<a>` tag got rendered correctly, but I guess this didn't work.

Reply:
I can still push commits, if you want me to. Your call!

Reply:
OK, please push a change `\[PDF\]` to `(PDF)`.

Reply:
On it. I'll amend the previous commit so it's a single commit.